datafusion/sqllogictest/test_files/explain.slt

```diff
@@ -242,6 +242,7 @@ physical_plan after OutputRequirements DataSourceExec: file_groups={1 group: [[W
 physical_plan after LimitAggregation SAME TEXT AS ABOVE
```

Comment:
I am confused about why YieldStreamExec does not appear in more of the explain `slt` plans

Reply:
Because, we added the built-in leaf nodes which will have built-in YieldStream, so the rule will ignore those cases. And almost all cases should have built-in YieldStream currently, so it will not add extra YieldStreamExec.

But if customer use custom leaf operator node, it will automatically add the YieldStreamExec, this is expected.